app/models/state_file_md_intake.rb

```diff
@@ -211,6 +211,22 @@ def address
     end
   end
 
+  def filer_1099_rs(primary_or_spouse)
+    state_file1099_rs.filter do |state_file_1099_r|
+      state_file_1099_r.recipient_ssn == send(primary_or_spouse).ssn
+    end
+  end
+
+  def sum_1099_r_followup_type_for_filer(filer_1099_rs, followup_type)
+    filer_1099_rs.sum do |state_file_1099_r|
+      if state_file_1099_r.state_specific_followup&.send(followup_type)
+        state_file_1099_r.taxable_amount&.round
+      else
+        0
+      end
+    end
+  end
+
```

Comment:
these methods were added to the intake b/c other calculators will need this logic as well